extensions/ql-vscode/src/databases/db-manager.ts

```diff
@@ -76,10 +76,24 @@ export class DbManager {
   }
 
   public async addNewRemoteRepo(nwo: string): Promise<void> {
+    if (nwo === "") {
+      throw new Error("Repository name cannot be empty");
+    }
+    if (this.dbConfigStore.doesRemoteDbExist(nwo)) {
+      throw new Error(`The repository '${nwo}' already exists`);
+    }
```

Comment:
We do this check twice now - is it really necessary? I think it would be better to decide who should take care of the validation and do the check only once.

Reply:
I don't understand this, check three times and throw errors three times? :o